R/subscript-loc.R

```diff
@@ -158,16 +158,21 @@ vec_as_location2_result <- function(i,
                                     missing,
                                     negative,
                                     arg) {
-  allow_missing <- arg_match(missing, c("error", "propagate")) == "propagate"
-  allow_negative <- arg_match(negative, c("error", "ignore")) == "ignore"
+  allow_missing <- (missing == "propagate")
```

Comment:
This becomes a length > 1 vector so we need to match?

Reply:
Let's just use `arg_match0()`? I'm going to focus on rlang and vctrs releases today and tomorrow.

Reply:
Sure, once rlang > 0.4.6 is on CRAN we can use that.